src/ddo/DDO.ts

```diff
@@ -118,7 +129,7 @@ export class DDO {
 
   /**
    * Finds a service of a DDO by index.
-   * @param index - index.
+   * @param index - index of the service in the DDO.
    * @returns Service.
    */
   public findServiceById<T extends ServiceType>(index: number): Service<T> {
```

Comment:
Should we rename this to get `findServiceByIndex`?

Reply:
:+1: